specification/mediaservices/resource-manager/Microsoft.Media/Accounts/stable/2023-01-01/Accounts.json

```diff
@@ -509,6 +509,39 @@
           "description": "The Private Endpoint Connections created for the Media Service account.",
           "type": "array",
           "readOnly": true
+        },
+        "minimumTlsVersion": {
+          "type": "string",
+          "enum": [
+            "Tls10",
+            "Tls11",
+            "Tls12",
+            "Tls13"
```

Comment:
Since they're strings, you could call them 1.0 or 1_0 to make it obvious they're not really 10... (in case there are ever that many versions of TLS?!)

Reply:
I think other services have probably done it this same way though.

Reply:
Actually I think for redisEnterprise since the property name was minimumTlsVersion, TLS is kind of implicit so I think we might have just called them "1.0" or "1_0".

Reply:
Want to crib from [https://github.com/Azure/azure-rest-api-specs/blob/891dd18a70057c2fee388573117683e6d0081bda/specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json#L1320](https://github.com/Azure/azure-rest-api-specs/blob/891dd18a70057c2fee388573117683e6d0081bda/specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json#L1320)

?

Reply:
It seems that several other teams are also using the syntax we have:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.management.cdn.models.minimumtlsversion?view=azure-dotnet
https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.management.storage.models.minimumtlsversion?view=azure-dotnet

I'm also unsure how 1.0 translates to c# as there cannot be periods in names to my knowledge. I will keep at is.

Reply:
I think the SDK code generation has ways to handle that :)